hy/compiler.py

```diff
@@ -1737,10 +1737,35 @@ def compile_require(self, expression):
         "unimport" it after we've completed `thing' so that we don't pollute
         other envs.
         """
-        expression.pop(0)
-        for entry in expression:
-            __import__(entry)  # Import it fo' them macros.
-            require(entry, self.module_name)
+        for entry in expression[1:]:
```

Comment:
It seems kind of odd that `require` and `import` handle a pretty much identical syntax in completely different ways. Would it somehow be possible to somehow merge the two?